charts/prefect-server/README.md

```diff
@@ -227,3 +228,5 @@ prefect-server:
 | serviceAccount.create | bool | `true` | specifies whether a ServiceAccount should be created |
 | serviceAccount.name | string | `""` | the name of the ServiceAccount to use. if not set and create is true, a name is generated using the common.names.fullname template |
 
+----------------------------------------------
+Autogenerated from chart metadata using [helm-docs v1.14.2](https://github.com/norwoodj/helm-docs/releases/v1.14.2)
```

Comment:
Wonder why `helm-docs` only includes this sometimes. Btw, since this project has a `.mise.toml` you can do `mise install` and it'll adhere to the version we specify there.